src/AutoMapper.EF6/EntityFrameworkExtensions.cs

```diff
@@ -432,4 +432,13 @@ public static IQueryable<TDestination> ProjectToQueryable<TDestination>(this IQu
             return queryable.ProjectTo<TDestination>(parameters).Decompile();
         }
     }
+#if !NETFRAMEWORK
+  internal static class InternalQueryableExtensions
+  {
+    internal static IQueryable<TDestination> Decompile<TDestination>(this IQueryable<TDestination> queryable)

```

Comment:
What's this?

Reply:
net standard does not have Decompile method only DecompileAsync